src/components/AdminFilterBusinesses/AdminFilterBusinesses.jsx

```diff
@@ -10,81 +9,88 @@ const AdminFilterBusinesses = () => {
   // Created the use states
   const { backend } = useBackend();
   //const [donationData, setDonationData] = useState([]);
-  const [businessDictionary, setBusinessDictionary] = useState([]);
   const [currentBusinessNum, setCurrentBusinessNum] = useState(0);
   const [currentPageNum, setCurrentPageNum] = useState(1);
-  const [currentTab, setCurrentTab] = useState('Active'); //change to all once all page is implemented
+  const [currentTab, setCurrentTab] = useState('All');
   const [pageLimit, setPageLimit] = useState(1);
+  const [searchTerm, setSearchTerm] = useState('');
 
   useEffect(() => {
-    const getData = async () => {
-      try {
-        loadInfo(currentTab);
-        console.log(businessDictionary);
-      } catch (error) {
-        console.error('Error fetching data:', error);
-      }
-    };
-    getData();
-  }, [currentTab, currentPageNum]);
+    try {
+      loadInfo(currentTab, searchTerm);
+    } catch (error) {
+      console.error('Error fetching data:', error);
+    }
+  }, [currentTab, currentPageNum, searchTerm]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'loadInfo'. Either include it or remove the dependency array.